doc/api/events.md

```diff
@@ -574,6 +574,15 @@ to indicate an unlimited number of listeners.
 
 Returns a reference to the `EventEmitter`, so that calls can be chained.
 
+### emitter.wrappedListeners(eventName)
+<!-- YAML
+added: REPLACEME
+-->
+- `eventName` {any}
+
+Returns a copy of the array of listeners for the event named `eventName`,
```

Comment:
I think saying 'event named' sounds a little confusing. Maybe something like 'event in' instead? Or maybe just 'event'?

Reply:
I can change this but we have another 9 instances of this exact phrasing in this doc.